.semaphore/semaphore.yml

```diff
@@ -1,5 +1,5 @@
 version: v1.0
-name: Performous Build on Ubuntu 18.04
+name: Semaphore 2.0 build
```

Comment:
I'd change it to `Performous build` since we're not really building semaphore 2.0 :P

Reply:
Now you're just being an ass 😂. You could probably fix that from within github even; I don't mind if you do because I'm not my phone.

Reply:
```suggestion
name: Performous build
```